osf/migrations/0121_remove_waffle_flags.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-07-23 13:16
+from __future__ import unicode_literals
+from waffle.models import Flag
+from django.db import migrations
+
+
+EMBER_WAFFLE_PAGES = [
+    'dashboard',
+    'home',
+]
+
+def format_ember_waffle_flag_name(page):
+    return 'ember_{}_page'.format(page)
+
+def add_ember_waffle_flags(state, schema):
+    for page in EMBER_WAFFLE_PAGES:
+        Flag.objects.get_or_create(name=format_ember_waffle_flag_name(page), everyone=False)
+    return
+
+def remove_waffle_flags(state, schema):
+    pages = [format_ember_waffle_flag_name(page) for page in EMBER_WAFFLE_PAGES]
+    Flag.objects.filter(name__in=pages).delete()
```

Comment:
I suggested in #8557 that these migrations might not be necessary, since there's no migration that creates them and therefore no guarantee that they exist.

That said, this won't error, so it's not a huge deal. And I suppose it saves a tiny bit of manual labor.